variables.tf

```diff
@@ -226,6 +226,12 @@ variable "trusted_signers" {
   description = "The AWS accounts, if any, that you want to allow to create signed URLs for private content. 'self' is acceptable."
 }
 
+variable "trusted_key_groups" {
+  type        = list(string)
+  default     = []
+  description = "A list of key group IDs that CloudFront can use to validate signed URLs or signed cookies."
+}
```

Comment:
Missing `variable "trusted_signers" {` statement

Reply:
Did you said this line?
https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn/pull/168/files/f10bb0ede35a82ca2b34314c2704f2ef42a54cb7#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eR223

Reply:
@nitrocode Can you response to me?

Reply:
Oh I see, I didn't realize it was already there.